src/braket/analog_hamiltonian_simulator/rydberg/validators/capabilities_constants.py

```diff
@@ -0,0 +1,21 @@
+from decimal import Decimal
+
+from pydantic.main import BaseModel
+
+
+class CapabilitiesConstants(BaseModel):
+
+    DIMENSIONS = 2
+    BOUNDING_BOX_SIZE_X: Decimal
+    BOUNDING_BOX_SIZE_Y: Decimal
+    MAX_TIME: Decimal
+    GLOBAL_AMPLITUDE_VALUE_MIN: Decimal
+    GLOBAL_AMPLITUDE_VALUE_MAX: Decimal
+    GLOBAL_DETUNING_VALUE_MIN: Decimal
+    GLOBAL_DETUNING_VALUE_MAX: Decimal
+
+    LOCAL_MAGNITUDE_SEQUENCE_VALUE_MIN: Decimal
+    LOCAL_MAGNITUDE_SEQUENCE_VALUE_MAX: Decimal
+
+    MAGNITUDE_PATTERN_VALUE_MIN: Decimal
+    MAGNITUDE_PATTERN_VALUE_MAX: Decimal
```

Comment:
Why all caps?

Reply:
Because they are all constants, should I change them? Also, in the previous round of reviewing, we talked about the possibility of moving this class to the schema, so maybe we can change them at that time.

Reply:
Generally global constants are all caps, but class variables follow normal conventions. I'm ok postponing renaming, but also wouldn't hurt, either way